db/migrate/20161201211657_add_intent_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddIntentToUsers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :users, :intent, :integer
```

Comment:
I was going to comment about whether we wanted to set `null: false`, but then I realized it will make our reports of who submitted intent much easier. We just have to remember to reset intent every year as part of the setup process. I'll open an issue for that as well.